Src/BootCamp.Chapter/Solution.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+
+public class Solution
+{
+    public int MyAtoi(string str)
```

Comment:
This code is extremely hard to read. We learned in chapter 1 that we should split big functions into smaller ones, where did all that go?